services/brig/schema/src/V60.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V60
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 60 "Add team_invitation_email table" $
+    schema'
+      [r|
+        CREATE TABLE team_invitation_email
+            ( email text PRIMARY KEY
+            , team set<uuid>
+            , id uuid
```

Comment:
```suggestion
            , user uuid
```

or `user_id`, or even `uid`, don't care.  just something indicating that yes dear reader, your hunch is correct, this is the user id.

Reply:
but it's not in this case, it's the invitation id. There's no user created yet at this point. I'll add a comment to the file.

Reply:
no, this is proof it needs a better name, not a comment!  please call it `invitation`, then.

Reply:
you're not calling `teams` `ids` either, right?

Reply:
ok ok! :) I'll rename.